### PR TITLE
flamenco: update advance nonce fee collector

### PR DESCRIPTION
**Pull request by kbhargava-jump**
agave uses the previous lamports per signature for advance nonce fee collection

### CODE REVIEW

**Review by ibhatt-jumptrading — APPROVED** [review]

**Review by mjain-jump — APPROVED** [review]
nice ty

File: src/flamenco/runtime/sysvar/fd_sysvar_fees.c [hunk 1]
```diff
@@ -88,6 +88,12 @@ fd_sysvar_fees_new_derived(
     me.max_lamports_per_signature = me.target_lamports_per_signature;
   }
 
+  if ( FD_UNLIKELY( slot_ctx->slot_bank.lamports_per_signature == 0 ) ) {
```
Comment (ibhatt-jumptrading) [inline src/flamenco/runtime/sysvar/fd_sysvar_fees.c:91]: if(

File: src/flamenco/runtime/fd_runtime.c [hunk 2]
```diff
@@ -70,6 +70,7 @@ fd_runtime_init_bank_from_genesis( fd_exec_slot_ctx_t *  slot_ctx,
 
   slot_ctx->slot_bank.fee_rate_governor = genesis_block->fee_rate_governor;
   slot_ctx->slot_bank.lamports_per_signature = 5000;
+  slot_ctx->prev_lamports_per_signature = 5000;
```
Comment (ibhatt-jumptrading) [inline src/flamenco/runtime/fd_runtime.c:73]: nit: 5000UL

**Comment by ibhatt-jumptrading** [comment]
lgtm

**Review by ibhatt-jumptrading — APPROVED** [review]

**Pull request merged**